internal/groups/http.go

```diff
@@ -0,0 +1,685 @@
+package groups
```

Comment:
Consider moving this to a separate `http` or `api` package (`/internal/groups/http` or `/internal/groups/api`) and breaking this file to the usual Mainflux API package structure (endpoints.go, requests.go, responses.go...).